R/formula-tools.R

```diff
@@ -247,7 +249,7 @@ parse_formula <- function(formula) {
 rw <- function(time, by, type = c("independent", "dependent")) {
   type <- match.arg(type)
   if (missing(time)) {
-    stop("time must be present")
+    rlang::abort("time must be present")
```

Comment:
Aren't missing variables reported as such? I'm a bit wary of providing a custom error message that doesn't actually say anything more than the base behavior would capture.